client/components/chat-list.jsx

```diff
@@ -3,14 +3,19 @@ import React from 'react';
 function ChatListItem(props) {
   return (
     <li className="chat-list-item">
-      <h2>{props.roomTitle}</h2>
+      <a href={`#rooms?roomId=${props.roomId}`}>
+        <h2>{props.roomTitle}</h2>
+      </a>
     </li>
   );
 }
 
 export default function ChatList(props) {
-  const allRooms = props.rooms.map(room => {
-    return <ChatListItem key={room.id} roomTitle={room.name}/>;
+  const sortedRooms = props.rooms.sort((a, b) => {
+    return b.id - a.id;
+  });
```

Comment:
It's better to have the server send you the information in the right order. You can `order by` in your SQL query.